src/test/scala/scalismo/faces/image/ImageIOTests.scala

```diff
@@ -68,15 +87,15 @@ class ImageIOTests extends FacesTestSuite {
     PixelImageIO.read[A](is).get
   }
 
-  describe("A random RGB color image") {
+  describe("A random sRGB color image") {
     it("survives a write-read cycle unaltered") {
-      diffOfIdentityTransform(img,writeReadCycle[RGB]) should be <= tolerance
+      diffOfsRGBIdentityTransform(img.map(_.tosRGB),writeReadCycle[sRGB]) should be <= tolerance
```

Comment:
Why do we need a tolerance here? Cant we construct/use some colors that are not altered by a read-write cycle?